climada_petals/hazard/test/test_tc_tracks_forecast.py

```diff
@@ -100,6 +105,20 @@ def test_ecmwf_multimessage(self):
             np.array(['70E', '70W', '71E', '71W', '72W'], dtype=str)
             )
 
+    def test_ecmwf_multimessage_missing_timeperiod(self):
+        """Test ECMWF reader should continue reading messages if one track misses timePeriod"""
+        with self.assertLogs('climada_petals.hazard.tc_tracks_forecast', level='INFO') as cm:
+            forecast = TCForecast()
+            forecast.fetch_ecmwf(files=TEST_BUFR_FILE_MULTIMESSAGE_MISSING_TIMEPERIOD)
+        assert ("Track 07L has no defined timePeriod. Track is discarded.") in cm.output[0]
```

Comment:
Use unittest assertion
```suggestion
        self.assertIn(
            "Track 07L has no defined timePeriod. Track is discarded.", cm.output[0]
        )
```